### PR TITLE
Feature/tweets

### PR DESCRIPTION
**Pull request by alitariq95**

### CODE REVIEW

**Comment by stakater-user** [comment]
@alitariq95 Yikes! You better fix it before anyone else finds out! [Build 1](https://jenkins-delivery.stakater.com/job/Stakater-lab/job/agile-integration-demo/job/PR-5/1/) has Failed!

**Comment by stakater-user** [comment]
@alitariq95 Yikes! You better fix it before anyone else finds out! [Build 2](https://jenkins-delivery.stakater.com/job/Stakater-lab/job/agile-integration-demo/job/PR-5/2/) has Failed!

**Comment by stakater-user** [comment]
@alitariq95 Yikes! You better fix it before anyone else finds out! [Build 3](https://jenkins-delivery.stakater.com/job/Stakater-lab/job/agile-integration-demo/job/PR-5/3/) has Failed!

**Comment by stakater-user** [comment]
@alitariq95 Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater-lab/gac-apache-camel-integration:0.0.0-PR-5-4-SNAPSHOT`

**Comment by stakater-user** [comment]
@alitariq95 Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater-lab/gac-apache-camel-integration:0.0.0-PR-5-5-SNAPSHOT`

**Comment by stakater-user** [comment]
@alitariq95 Yikes! You better fix it before anyone else finds out! [Build 6](https://jenkins-delivery.stakater.com/job/Stakater-lab/job/agile-integration-demo/job/PR-5/6/) has Failed!

**Comment by stakater-user** [comment]
@alitariq95 Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater-lab/gac-apache-camel-integration:0.0.0-PR-5-7-SNAPSHOT`

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -1 +1,7 @@
+kafka.address=${KAFKA_ADDRESS:localhost}
+kafka.port=${KAFKA_PORT:9092}
 
+twitter.consumerKey=NMqaca1bzXsOcZhP2XlwA
+twitter.consumerSecret=VxNQiRLwwKVD0K9mmfxlTTbVdgRpriORypnUbHhxeQw
+twitter.accessToken=26693234-W0YjxL9cMJrC0VZZ4xdgFMymxIQ10LeL1K8YlbBY
+twitter.accessTokenSecret=BZD51BgzbOdFstWZYsqB5p5dbuuDV12vrOdatzhY4E
```
Comment (usamaahmadkhan) [inline src/main/resources/application.properties:7]: This is a public repository, committing a access token in a public repo can be dangerous. Get these from env as well

**Pull request merged**